spec/visual/samples/clean

```diff
@@ -0,0 +1,224 @@
+implementation module Data.GenCompress
+
+// This sample file was taken from Clean Platform:
```

Comment:
Real world samples can be helpful but a fair amount of the code here seems to be duplicative (from a testing point of view). Would it be possible to reduce some of that so that each line is more adapted to testing rules in the lexer?